docs/FEEDBACK.md

```diff
@@ -0,0 +1,95 @@
+# NearIt-UI for feedback request pop-up
+#### Basic example
+If you want your app to display a feedback request in a beautiful pop-up dialog, use this simple code:
+
+```java
+ // ...
+ startActivity(
+         NearITUIBindings.getInstance(YourActivity.this)
+            .createFeedbackIntentBuilder(feedback)
+            .build());
+```
+
+where, `feedback` is an instance of NearIT SDK `Feedback` class. Further information on coupons and other in-app content can be found [here](http://nearit-android.readthedocs.io/en/latest/in-app-content/).
+
```

Comment:
Add "The Feedback UI also takes care of delivering the user response to the SDK library and showing the proper success or failure status of the user action"

Reply:
done